### PR TITLE
Update license

### PR DESCRIPTION
**Pull request by jperon**
Just a suggestion. If you’d better use Apache 2 (like other parts), no problem for me: MIT is a bit more permissive.

### CODE REVIEW

**Review by paulfitz — COMMENTED** [review]
Thanks @jperon. Generous to transfer copyright. I'd accept this if combined with an update to the README to make clear who deserves credit for developing this :)

**Comment by jperon** [comment]
Here you are :-)

File: Readme.md [hunk 1]
```diff
@@ -38,7 +38,14 @@ will make development environment ready: `docker run -it -v $PWD:/src pug_py`.
...
 
+# Credits
+
+@berhalak for his [fiddle widget](https://github.com/berhalak/my-widgets/tree/main/fiddle), that was the basis of this one.
+
+@jperon for adaptation to pug and python.
 
 # License
 
+V. LICENSE
```
Comment (paulfitz) [inline Readme.md:49]: What does this mean?

**Review by paulfitz — COMMENTED** [review]
Credit section looks good, thank you. What does the V. LICENSE line mean?

File: Readme.md (hunk 1 above)
Comment (jperon) [reply Readme.md]: *vide*. It can be used in French instead of *cf*...

**Review by paulfitz — APPROVED** [review]
Thanks @jperon !

**Pull request merged**